src/query_ctrl.js

```diff
@@ -24,6 +25,7 @@ export class BitmovinAnalyticsDatasourceQueryCtrl extends QueryCtrl {
 
     this.metrics = AGGREGATION_LIST;
     this.fields = ATTRIBUTE_LIST;
+    this.groupByFields = GROUP_BY_ATTRIBUTE_LIST;
     this.orderByFields = ORDERBY_ATTRIBUTES_LIST;
     if (this.datasource.isAdAnalytics === true) {
       this.fields = AD_ATTRIBUTE_LIST;
```

Comment:
```suggestion
      this.fields = AD_ATTRIBUTE_LIST;
      this.groupByFields = GROUP_BY_AD_ATTRIBUTE_LIST;
```

Reply:
The datasource can either query `ads` or normal data. Adapt the groupBy fields based on that